src/time_object.cpp

```diff
@@ -8,7 +8,16 @@ TimeObject *TimeObject::at(Env *env, Value time, Value subsec, Value unit) {
         auto scale = convert_unit(env, unit ? unit : "microsecond"_s);
         rational = rational->add(env, convert_rational(env, subsec)->div(env, scale))->as_rational();
     }
-    return TimeObject::create(env, rational, Mode::Localtime);
+    return create(env, rational, Mode::Localtime);
+}
+
+TimeObject *TimeObject::at(Env *env, Value time, Value subsec, Value unit, Value in) {
+    auto result = at(env, time, subsec, unit);
+    if (in) {
+        result->m_time.tm_gmtoff = normalize_timezone(env, in);
+        result->m_time.tm_zone = (char *)"UTC";
```

Comment:
Does this `(char *)` cast need to be there? According to [this doc](https://www.gnu.org/software/libc/manual/html_node/Broken_002ddown-Time.html), the field takes a `const char *` anyway.

Reply:
Ah yes, that recast was unnecessary. Removed it, reran the spec locally and am re-pushing through the CI.

Reply:
Weird, apparently it is needed on macos. Sorry! I wonder if we should strdup it then? But then we have to free it later? Gross.

Reply:
I tested locally with Linux and didnt need it, didn't occur to me that macos would be different here, but I should know better at this point to take anything for granted across platforms.
I don't have enough C++ experience to know the pros and cons of the non-const re-cast, though I tend to avoid malloc/free whenever I can 😸

Reply:
I pushed a commit to hopefully fix this without a cast to remove const.